src/target/riscv/riscv_reg.h

```diff
@@ -21,6 +21,16 @@ void riscv_reg_free_all(struct target *target);
 
 /** Write all dirty registers to the target. */
 int riscv_reg_flush_all(struct target *target);
+/**
+ * Checks whether there are dirty registers, reprorting dirty registers usign
+ * the supplied "log_level".
+ */
```

Comment:
Detail - typo fix and a minor reword to improve the clarity:

```suggestion
/**
 * Check whether there are any dirty registers in the OpenOCD's register cache.
 * In addition, all dirty registers will be reported to the log using the supplied "log_level".
 */
```

Reply:
Addressed.